src/pages/Checkout/CheckoutPage.vue

```diff
@@ -710,11 +716,39 @@ export default {
 }
 
 .empty-basket {
+	position: relative;
 	margin: 0 auto;
-}
 
-.empty-basket-heading {
-	font-weight: 500;
+	.empty-basket-heading {
+		font-weight: 500;
+	}
+
+	.empty-basket-loans {
+		position: relative;
+		min-height: 23rem;
+
+		#updating-overlay {
```

Comment:
these styles fix the loader layout to work in a contained context. Also had to do this with the interstitial so we'll likely move this into the LoadingOverlay component soon.

Reply:
✔️